Analysis/DEEP2_R23_O32/individual_detect.py

```diff
@@ -25,7 +25,7 @@
 #from . import general
 
 import Metallicity_Stack_Commons
-from Metallicity_Stack_Commons.analysis.composite_indv_detect import main
+from Metallicity_Stack_Commons.Metallicity_Stack_Commons.analysis.composite_indv_detect import main
```

Comment:
This is a bit confusing as if there should not be an extra Metallicity_Stack_Commons